src/styles/base/_reset.scss

```diff
@@ -0,0 +1,21 @@
+html,
+body,
+div,
+h1,
+h2,
+h3,
+p,
+span,
+ul,
+li,
+button {
```

Comment:
The selector list includes 'button', which does not typically have a 'list-style' property. Consider removing 'list-style: none;' for better specificity.